ntbs-integration-tests/NotificationPages/OverviewPageTests.cs

```diff
@@ -183,6 +183,26 @@ public async Task OverviewPageShowsCreateNewNotificationForThisPatientButton_IfN
             // Assert
             Assert.NotNull(document.QuerySelector("#new-linked-notification-button"));
         }
+
+        [Fact]
+        public async Task OverviewPageHidesPrintOverviewButton_IfNotificationIsDraft()

```

Comment:
As this is rerouting from overview page to patient page to then show no print overview button it might be nice to be explicit in following the redirect and then checking the button doesn't exist. The name is slightly confusing as the overview page doesn't exist for a draft

Reply:
Upon further contemplation this doesn't add anything as we already have a test that drafts don't have an overview page